src/Summoner/Project.hs

```diff
@@ -198,3 +133,23 @@ generateProject projectName Targets{..} = do
         "git" ["add", "."]
         "git" ["commit", "-m", "Create the project"]
         "git" ["push", "-u", "origin", "master"]
+
+    categoryText :: Text
```

Comment:
Cool idea to move this under `where`. 
You get the joke?) under `where` is like _underwear_ :joy: